OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -202,6 +325,7 @@ Properties trackVideoOrientation(String videoId, Double currentTime,
         String VIDEO_DOWNLOADED = "edx.bi.video.downloaded";
         String USERLOGOUT = "edx.bi.app.user.logout";
         String USERLOGIN = "edx.bi.app.user.login";
+        String USER_REGISTER = "edx.bi.app.user.register";
```

Comment:
Note that the current implementation seems to be treating login and registration inconsistently, in that the login events are sent when we receive a successful response from the server, but we only track the button click on registration. This is probably what this was introduced to resolve.